packages/renderer/src/lib/network/NetworkDetails.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+import { StatusIcon, Tab } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+
+import { networksListInfo } from '/@/stores/networks';
+
+import Route from '../../Route.svelte';
+import VolumeIcon from '../images/VolumeIcon.svelte';
+import DetailsPage from '../ui/DetailsPage.svelte';
+import { getTabUrl, isTabSelected } from '../ui/Util';
+import { NetworkUtils } from './network-utils';
+import NetworkActions from './NetworkActions.svelte';
+import NetworkDetailsInspect from './NetworkDetailsInspect.svelte';
+import NetworkDetailsSummary from './NetworkDetailsSummary.svelte';
+import type { NetworkInfoUI } from './NetworkInfoUI';
+
+interface Props {
+  networkName: string;
+  engineId: string;
+}
+
+let { networkName, engineId }: Props = $props();
+
+let network: NetworkInfoUI | undefined = $state();
```

Comment:
suggestion(non-blocking): we could maybe remove the intermediate state `network` as it could be derived from `matchingNetwork`

```ts

let network = $derived.by(() => {
  const matchingNetwork = $networksListInfo.find(
    network => network.Name === networkName && network.engineId === engineId,
  );
  if (matchingNetwork) {
    return networkUtils.toNetworkInfoUI(matchingNetwork);
  }
});

$effect(() => {
  if (!network && detailsPage) {
    detailsPage.close();
  }
});
```